search.py

```diff
@@ -406,102 +406,75 @@ def astar_search(problem, h=None):
 
 class EightPuzzle(Problem):
 
-    """The problem of sliding tiles numbered from 1 to 8 on a 3x3 board,
+    ''' The problem of sliding tiles numbered from 1 to 8 on a 3x3 board,
```

Comment:
No. Pep 257: "For consistency, always use """triple double quotes""" around docstrings."